containers/doc/server-kubernetes/README.md

```diff
@@ -28,143 +30,144 @@ Run this commant to watch it restart:
 
 ```
 k3s ctr images import server-image.tar
 ```
 
-In order to tell k3s to not pull the image, add `imagePullPolicy: Never` to all `initContainer`s and `container` in the `server.yaml` file:
+In order to tell k3s to not pull the image, set the `pullPolicy` helm chart value to `Never` by passing a `--helm-value=values.yaml` parameter to `uyuniadm` with the following `values.yaml` file content:
 
 ```
-sed 's/^\( \+\)image:\(.*\)$/\1image: \2\n\1imagePullPolicy: Never/' -i server.yaml
+pullPolicy: Never
 ```
 
-### Migrating from a regular server
+## For rke2
 
-Stop the source services:
+RKE2 doesn't allow to preload images on the nodes.
+Instead, use `skopeo` to import the images in a local registry and use this one to install.
 
+⚠️  **TODO** Prepare instructions
 ```
-spacewalk-service stop
-systemctl stop postgresql
+# TODO copy the cert-manager and uyuni helm charts
+# TODO Copy the cert-manager and uyuni images
+# TODO Install cert-manager from the local chart
+# TODO Set the uyuniadm --image parameter
 ```
 
-Create a password-less SSH key and create a kubernetes secret with it:
+## For podman
+
+With k3s it is possible to preload the container images and avoid it to be fetched from a registry.
+For this, on a machine with internet access, pull the image using `podman`, `docker` or `skopeo` and save it as a `tar` archive.
+For example:
 
 ```
-ssh-keygen
-kubectl create secret generic migration-ssh-key --from-file=id_rsa=$HOME/.ssh/id_rsa --from-file=id_rsa.pub=$HOME/.ssh/id_rsa.pub
+podman pull registry.opensuse.org/systemsmanagement/uyuni/master/servercontainer/containers/uyuni/server:latest
+podman save --output server-image.tar registry.opensuse.org/systemsmanagement/uyuni/master/servercontainer/containers/uyuni/server:latest
 ```
-Add the generated public key to the server to migrate authorized keys.
 
-Run the migration job:
+or
 
 ```
-kubectl apply -f migration-job.yaml
+skopeo copy docker://registry.opensuse.org/systemsmanagement/uyuni/master/servercontainer/containers/uyuni/server:latest docker-archive:server-image.tar:registry.opensuse.org/systemsmanagement/uyuni/master/servercontainer/containers/uyuni/server:latest
 ```
 
-To follow the progression of the process, check the generated container log:
+Transfer the resulting `server-image.tar` to the server and load it using the following command:
 
 ```
-kubectl logs (kubectl get pod -ljob-name=uyuni-migration -o custom-columns=NAME:.metadata.name --no-hea
-ders)
+podman load -i server-image.tar
 ```
 
-Once done, both the job and its pod will remain until the user deletes them to allow checking logs.
-
-Proceed with the next steps.
+# Migrating from a regular server
 
-***Hostname***: this procedure doesn't handle any hostname change.
-Certificates migration also needs to be documented, but that can be guessed for now with the instructions to setup a server from scratch.
+The migration procedure does not perform any hostname rename.
```

Comment:
I think you should mention in here that we cannot perform an in-place migration, and a new machine is needed.